widgets/visualization/org.eclipse.nebula.visualization.xygraph/src/org/eclipse/nebula/visualization/xygraph/linearscale/LinearScaleTicks.java

```diff
@@ -475,6 +490,11 @@ private void updateTickLabelForLinearScale(double min, double max, int length, d
 			tickLabelPositions.add(tickLabelPosition);
 		}
 
+		if (scale.isHorizontal()) {
+			System.err.println("For length of " + length);
```

Comment:
I do not know the logging philosophy in Nebula. Do you think this code is relevant ? Should it be removed ?

Reply:
No, accidentally left in debugging.